ci/py310_sgeop-oldest.yaml

```diff
@@ -0,0 +1,26 @@
+name: py310_sgeop-oldest
+channels:
+  - conda-forge
+dependencies:
+  - python=3.10
+  - esda=2.5.1
+  - fiona=1.9
+  - geopandas=0.13.0
+  - git
+  - libpysal
+  - momepy
+  - networkx
+  - numpy
+  - pandas
+  - pip
+  - pyarrow
+  - pyogrio
+  - scipy
+  - shapely
+  # testing
+  - pytest
+  - pytest-cov
+  - pytest-xdist
+  - pip:
+    # from bleeding edge functionality
+    - git+https://github.com/pysal/momepy.git
```

Comment:
That is due to https://github.com/pysal/momepy/pull/652 right? I will cut a patch 0.8.1 to pin that.

Reply:
https://github.com/pysal/momepy/releases/tag/v0.8.1